assets/js/script.js

```diff
@@ -0,0 +1,195 @@
+function openNav() {
+  document.getElementById('myNav').style.width = '100%';
+}
+
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+
+const closeBtn = document.getElementById('close-btn');
+const IconMenu = document.getElementById('click-image');
+const hamItem = document.getElementsByClassName('nav-list');
+
+closeBtn.addEventListener('click', () => {
+  closeNav();
+});
+IconMenu.addEventListener('click', () => {
+  openNav();
+});
+
+for (let index = 0; index < hamItem.length; index += 1) {
+  hamItem[index].addEventListener('click', () => {
+    closeNav();
+  });
+}
+
+const workDiv = document.querySelector('.work');
+
+const projects = [
```

Comment:
- Kindly provide the requested style by the design guidelines
![Screen Shot 2021-10-14 at 15 03 22](https://user-images.githubusercontent.com/22258274/137387631-8250ba06-395c-46b8-a7d0-1dfc515b7a57.png)